cli/crates/cli/tests/compilation_error.rs

```diff
@@ -5,23 +5,25 @@ use std::collections::HashMap;
 use backend::project::ConfigType;
 use serde_json::Value;
 use utils::consts::{
-    COMPILATION_ERROR_QUERY, COMPILATION_ERROR_RESOLVER_MUTATION, COMPILATION_ERROR_RESOLVER_SCHEMA,
+    COMPILATION_ERROR_QUERY, COMPILATION_ERROR_RESOLVER_QUERY, COMPILATION_ERROR_RESOLVER_SCHEMA,
     COMPILATION_ERROR_SCHEMA, DEFAULT_QUERY, DEFAULT_SCHEMA, ENVIRONMENT_SCHEMA,
 };
 use utils::environment::Environment;
 
 #[test]
-#[cfg_attr(target_os = "windows", ignore)]
-fn compilation_error() {
+fn compilation_error_schema() {
     let mut env = Environment::init();
     env.grafbase_init(ConfigType::GraphQL);
     env.write_schema(COMPILATION_ERROR_SCHEMA);
+
+    std::thread::sleep(std::time::Duration::from_secs_f64(0.5));
```

Comment:
Why is this needed? Would be best to use something like our schema polling to be able to continue as fast as possible rather than waiting a set amount of time